draft-ietf-mls-architecture.md

```diff
@@ -1864,13 +1864,6 @@ one-time, per client, authentication secret which can be exchanged between users
 to prove their identity to each other. This can be done for instance using a QR
 code that can be scanned by the other parties.
 
-Another way to improve the security for the users is to provide a transparency
-mechanism which allows each user to check if credentials used in groups have
-been published in the transparency log. Another benefit of this mechanism is for
-revocation. The users of a group could check for revoked keys (in case of
-compromise detection) using a mechanism such as CRLite or some more advanced
-privacy preserving technology.
```

Comment:
Add back some mention of revocation. Maybe:

"When the credential type support revocation, the users of a group could check for revoked keys, for example for `x509` credentials, using a mechanism such as CRLite."

(I don't particularly care if we mention CRLite or some other mechanism, but I think we should say something about revocation.)